spring-boot-cli/samples/template.groovy

```diff
@@ -2,6 +2,8 @@ package org.test
 
 import static org.springframework.boot.groovy.GroovyTemplate.*;
 
+@Grab("groovy-templates")
+
```

Comment:
Unfortunately, without this I'm getting an error when running this template:
Caused by: java.lang.NoClassDefFoundError: Unable to load class org.springframework.boot.groovy.GroovyTemplate due to missing dependency groovy/text/TemplateEngine

Adding this line seemed like the only way to make it run